CRM/Member/Form.php

```diff
@@ -78,6 +77,23 @@ class CRM_Member_Form extends CRM_Contribute_Form_AbstractEditPayment {
   public $_priceSetId;
```

Comment:
@litespeedmarc it doesn't appear that you have addressed @eileenmcnaughton 's concern here. Could you comment?